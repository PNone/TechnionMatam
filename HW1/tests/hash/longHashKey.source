barnkklasdfnalsdkfnsaldkfnlknlfkasndflknasfawerwaoeifjasdlknlasdknflkasdjfaa barnkklasdfnalsdkfnsaldkfnlknlfkasndflknasfawerwaoeifjasdlknlasdknflkasdjfaabarnkklasdfnalsdkfnsaldkfnlknlfkasndflknasfawerwaoeifjasdlknlasdknflkasdjfaabarnkklasdfnalsdkfnsaldkfnlknlfkasndflknasfawerwaoeifjasdlknlasdknflkasdjfaa 30 20:40
barnkklasdfnalsdkfnsaldkfnlknlfkasndflknasfawerwaoeifjasdlknlasdknflkasdjfaa barnkklasdfnalsdkfnsaldasdfsadfasdflkasdnfldsasdfasdfasdfkfnlknlfkasndflknasfawerwaoeifjasdlknlasdknflkasdjfaabarnkklasdfnalsdkfnsaldkfnlknlfkasndflknasfawerwaoeifjasdlknlasdknflkasdjfaabarnkklasdfnalsdkfnsaldkfnlknlfkasndflknasfawerwaoeifjasdlknlasdknflkasdjfaa 30 20:40
jvoaisdjfoasdfnaksdjnvalkwejofajosdjnfkajsdnvkasjdfnkasjdfnkasdjfnkasdjfhaskdjvnasdkjfnaskdfjnasdkjvnasdkjnasdkjfnakjeghoaivnlaksdnflaksdnfkajvnoawiefjpowqiemvlkdsmvlaksdnflaskdvnlaksdnflaksdfoawiefaoiwuevbaysivbakjsdvnkasdnlkajdsnfalkdjvnlaksdjfnbasdlfjkasjalkjasdifjasdnflkasdvjnaksjdvasdvjasdkvjbadvlkjbadsflkjbadsflkjadvnkladjvnlkasdjfhaoweifihiaeyfuadoyvbasldhvbaskdncsadklnjaslkvdjnlakfvjnasdfjkhaiufysadoivjbasdlkl regev 30 20:40