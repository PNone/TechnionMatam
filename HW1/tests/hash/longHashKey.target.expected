adcfe1032547698badcf
adcfe1032547698badcf
07214b658f243aad98fd